kujira/assetlist.json

```diff
@@ -2362,6 +2362,49 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nami.svg"
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+          "exponent": 0
+        },
+        {
+          "denom": "YUM.axl",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+      "name": "Axelar Wrapped YUM",
+      "display": "yum",
```

Comment:
Error: Display denomination yum is not defined as a denom_unit.
Then denom with exponent 18 is defined as YUM.axl